audformat/core/testing.py

```diff
@@ -186,6 +188,31 @@ def add_table(
     return db[table_id]
 
 
+def create_attachment_files(
```

Comment:
I guess this introduces a breaking change, since we now have to call this method before we can save a database created with `audformat.testing.create_db()`. But I guess there is no other solution unless we choose a table as attachment, which does not make much sense.

Reply:
One solution would be that we do not create an attachment with `audformat.testing.create_db()`, but that the user has to add one herself. This might be a cleaner solution?

Reply:
Yes, it might be safer, but it means we will not have an `Attachment` entry in the default database, which is also not nice. I'm undecided what's the better way to go :)

Reply:
Maybe it's not so bad to introduce a breaking change here as it will probably only affect `audb`, where we anyway need to add support for `Attachment` now.

Reply:
It should also not affect the user part of any package, but only the tests. At least I hope that we use `audformat.testing.create_db()` only for testing inside `audb`.

Reply:
Ok, let's stick with the implementation as is.